packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -36,25 +36,25 @@ if (-not $LaunchOnly) {
         }
     }
 
-    if ($null -eq $filter -or $filter -eq "Sample-Service") {
-      Write-Host "Generating SampleService" -ForegroundColor Cyan
-      $sampleDir = Join-Path $packageRoot '..' '..' 'docs' 'samples' 'client' 'csharp' 'SampleService'
+    # if ($null -eq $filter -or $filter -eq "Sample-Service") {
```

Comment:
For some reason, Sample-Service generation deleted all the csproj files in the generator, guess it was caused by the file reference. Skip this for now to focus on the tsp version bump.